tests/all/unit/api/helpers/test_notification.py

```diff
@@ -0,0 +1,146 @@
+import unittest
+from flask import current_app
+from app.api.helpers.notification import (
+    send_notif_new_session_organizer,
+    send_followup_notif_monthly_fee_payment,
+    send_notif_after_event,
+    send_notif_after_export,
+    send_notif_after_import,
+    send_notif_event_role,
+    send_notif_monthly_fee_payment,
+    send_notif_session_accept_reject,
+    send_notif_ticket_purchase_organizer,
+)
+import app.factories.common as common
+from app.factories.session import SessionFactory
+from app.factories.user import UserFactory
+from app.models.notification import Notification
+from tests.all.integration.utils import OpenEventTestCase
+
+
+class TestNotificationHelper(OpenEventTestCase):
+    def test_send_notif_new_session_organizer(self):
+        """Method to test new session notification"""
+        with self.app.test_request_context():
+            user = UserFactory()
+            test_session = SessionFactory()
+            link = common.url_
+            event_name = common.string_
+            current_app.config['TESTING'] = False
```

Comment:
Why?

Reply:
the send_notification isn't storing in db when app config is in TESTING mode

Reply:
Then find the reason